server/sonar-db-dao/src/test/java/org/sonar/db/qualitygate/QualityGateDbTester.java

```diff
@@ -41,12 +43,17 @@ public QualityGateDbTester(DbTester db) {
     this.dbSession = db.getSession();
   }
 
-  public QualityGateDto insertQualityGate() {
-    return insertQualityGate(randomAlphanumeric(30));
+  public QualityGateDto insertQualityGate(String name) {
+    return insertQualityGate(qualityGate -> qualityGate.setName(name));
   }
 
-  public QualityGateDto insertQualityGate(String name) {
-    QualityGateDto updatedUser = dbClient.qualityGateDao().insert(dbSession, new QualityGateDto().setName(name).setBuiltIn(false));
+  @SafeVarargs
+  public final QualityGateDto insertQualityGate(Consumer<QualityGateDto>... dtoPopulators) {
```

Comment:
I'm wondering is there a usage with multiple consumer, one is not enough ?

Reply:
The usage of var-args is only to allow calling this method without any argument. It could have been done by using 2 methods, but this pattern is already used a lot in XXXDbTester